jenkins/release.jenkinsFile

```
@@ -0,0 +1,16 @@
+lib = library(identifier: 'jenkins@1.5.3', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+standardReleasePipelineWithGenericTrigger(
+    tokenIdCredential: 'jenkins-opensearch-hadoop-generic-webhook-token',
```

Is the token created?

---

@gaiksaya is creating it now.

---

Since CI pipeline is blocked, I will be creating it manually maybe name it something else so later when it actually deploys we can use this name?
```suggestion
    tokenIdCredential: 'jenkins-opensearch-hadoop-generic-webhook-token-manual',
```